cli/script/command-executor.ts

```diff
@@ -283,6 +283,26 @@ function deploymentRename(command: cli.IDeploymentRenameCommand): Promise<void>
         });
 }
 
+function deploymentView(command: cli.IDeploymentViewCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+    var storedAppId: string;
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<string> => {
+            throwForInvalidAppId(appId, command.appName);
+            storedAppId = appId;
+
+            return getDeploymentId(appId, command.deploymentName);
+        })
+        .then((deploymentId: string): Promise<any>[] => {
+            throwForInvalidDeploymentId(deploymentId, command.deploymentName, command.appName);
+
+            return [ sdk.getDeploymentKeys(storedAppId, deploymentId), sdk.getPackageHistory(storedAppId, deploymentId) ];
```

Comment:
You don't need to fetch or display the deployment key.  "deployment list" already shows deployment keys.